src/common/util.js

```diff
@@ -167,6 +167,7 @@ function yyyy_mm_dd_to_epoch(yyyy_mm_dd, options) {
 
 /* format the number (1,234,567.89), source: http://stackoverflow.com/questions/2254185 */
 function formatPrice(float,currency) {
+    var minimumFractionDigits = currency.toUpperCase() === 'BTC' || currency.toUpperCase() === 'XBT' ? 8 : 2; //Because backend is confusing
```

Comment:
@4p00rv @ashkanx Is this not something backend should tell us if we are not going by the standards of 2 decimals for account currency?